views/explorer.tmpl

```diff
@@ -15,14 +15,16 @@
             {{$topBlock = ((index .Data 0).Height)}}
             {{$Offset := (subtract .BestBlock $topBlock)}}
             {{$pendingBlocks := ((index .Data 0).Height)}}
+            {{$rowsCount := (int64 $blocksCount)}}
             <div class="d-flex justify-content-between align-items-end">
                 <span class="h2 d-flex pt-4 pb-1 pr-2">
                     Blocks
                     <span class="dcricon-info fs24 ml-3 mt-2"></span>
                 </span>
                 <div class="pb-1 d-flex justify-content-end align-items-end flex-wrap">
                   <span class="fs12 nowrap text-secondary px-2 my-2">
-                      {{intComma (add $Offset 1)}} &ndash; {{intComma (add $Offset .Rows)}} of {{intComma (add .BestBlock 1) }} rows
+                      <!-- {{intComma (add $Offset 1)}} &ndash; {{intComma (add $Offset .Rows)}} of {{intComma (add .BestBlock 1) }} rows -->
+                      {{intComma (add $Offset 1)}} &ndash; {{intComma (add $Offset $rowsCount)}} of {{intComma (add .BestBlock 1) }} rows
```

Comment:
Can delete the comment. Consider adding a field to the template data instead of using a template variable.

Reply:
Okay

Reply:
I've made the change